packages/code-generator/src/plugins/component/shared/css-extractor.ts

```
@@ -0,0 +1,69 @@
+    moduleBuilder.addCssClass(className, styles);
+
+    // 4. 将生成的类名存入 irNode.css 字段，供模板/JSX 插件消费
+    irNode.css = className;
+
+    // 5. 清除 irNode.styles，防止模板/JSX 插件再次处理它
+    delete irNode.styles;
+  }
+
+  // 6. 递归遍历子节点
+  if (irNode.children && irNode.children.length > 0) {
+    irNode.children.forEach((child) =>
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)